core-bundle/assets/backend.js

```diff
@@ -49,8 +49,8 @@ const mooDomready = () => {
     if (!document.body.mooDomreadyFired) {
         document.body.mooDomreadyFired = true;
 
-        if (window.removeEvents) {
-            window.removeEvents();
+        if (Element.Events.removeEvents) {
```

Comment:
This condition is never `true` on my system (Safari and Firefox on macOS). I added `console.log(Element.Events.removeEvents);` above this line and it is always `undefined`.

Since this PR also fixes #8227 by removing `window.addEvent("domready", …)` in the files below, I'm all for merging it. But are we sure that we want to ignore that?

Reply:
Besides that, anything that would work such as `Element.removeEvents()`, `window.removeEvents()` or `document.body.removeEvents()` will remove the `domready` event and break all MooTools related stuff (like it is currently in Contao 5.5.8). So this basically only works because it is never executed.